### DISCUSSION TITLE
Sonoff ZB_BRIDGE LEDLINK not working

### DISCUSSION THREAD

**Discussion by jeroenst** [arendst/Tasmota#11091] [Q&A] [answered]
I want the led of my sonoff ZB_BRIDGE to be on when the device is working.

Currently when the device is powered on there it is not visible if the device is active, only when zigbee data is send or received. It looks like it's powered off by default.

I tried to change gpio13 in the zigbee template to ledlink but I think it's overridden by the template:

https://github.com/arendst/Tasmota/blob/6ce3e6be7c1943594ec29dd02e778416dc225973/tasmota/tasmota_template.h#L2222

How can I solve this problem?

**Comment by s-hadinger** [comment, marked as answer]
You can use a custom template similar to Module 75 and use LedLinki instead of LedLink. This will inverse the status of this led

**Reply by jeroenst** [reply]
Thank you, I have a few questions:
Why can't I override the setting in the template? 
Do I have to create a template from scratch?
Which template can I use best for my goal?

**Reply by s-hadinger** [reply]
Please ask these questions on Discord
